recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -370,6 +370,21 @@ def validate(self):
                 "Doesn't support gcc5 / shared. "
                 "See https://github.com/conan-io/conan-center-index/pull/4401#issuecomment-802631744"
             )
+        if (tools.Version(self.version) < "1.9.234"
+            and self.settings.compiler == "gcc"
+            and tools.Version(self.settings.compiler.version) >= "11.0"
+            and self.settings.build_type == "Release"):
+            raise ConanInvalidConfiguration(
+                "Versions prior to 1.9.234 don't support release builds on >= gcc 11 "
+                "See https://github.com/aws/aws-sdk-cpp/issues/1505"
+            )
+        if (self.settings.compiler == "clang"
+            and tools.Version(self.settings.compiler.version) >= "13.0"
+            and self.options.get_safe("text-to-speech")):
+            raise ConanInvalidConfiguration(
+                "Doesn't support 'text-to-speech' option with clang >= 13. "
+                "See https://github.com/aws/aws-sdk-cpp/issues/1816"
+            )
```

Comment:
You might add this commit as a patch instead of stopping clang >= 13 support: https://github.com/IstraResearch/aws-sdk-cpp/commit/54992e17020d1dfc979c26959e62b69e1b10921c

I have clang 13.1.6 so I'm unable to test your PR locally.

Reply:
Here is the tracking issue: https://github.com/aws/aws-sdk-cpp/issues/1816

Reply:
I wasn't sure if actual code changes is something we do in conan-center. But sure, the change is trivial and a real fix.
Will do